packages/eas-cli/src/commandUtils/flags.ts

```diff
@@ -0,0 +1,19 @@
+import { Flags } from '@oclif/core';
+
+export const EasNonInteractiveFlags = {
+  nonInteractive: {
+    json: Flags.boolean({
+      description: 'Enable JSON output, non-JSON messages will be printed to stderr.',
+      dependsOn: ['non-interactive'],
+    }),
+    'non-interactive': Flags.boolean({
+      description: 'Run the command in non-interactive mode.',
+    }),
+  },
+  jsonOnly: {
+    json: Flags.boolean({
+      description: 'Enable JSON output, non-JSON messages will be printed to stderr.',
+      dependsOn: ['non-interactive'],
```

Comment:
If jsonOnly is used there is no non-interactive flag to set.

Reply:
haha, must've slipped past me. nice catch!